components/core/src/main/java/com/bloxbean/cardano/yaci/store/core/metrics/MetricsService.java

```diff
@@ -17,13 +18,19 @@ public class MetricsService {
     public static final String YACI_STORE_CURRENT_ERA = "yaci.store.current.era";
     public static final String YACI_STORE_SYNC_MODE = "yaci.store.sync_mode";
     public static final String YACI_STORE_PROTOCOL_MAGIC = "yaci.store.protocol_magic";
+    public static final String YACI_STORE_CURRENT_SLOT = "yaci.store.current.slot";
+    public static final String YACI_STORE_CONNECTION_STATUS = "yaci.store.connection.status";
```

Comment:
Should me make it `yaci.store.node.connection.status` ? Same for `yaci.store.node.connection.resets`